### PR TITLE
Fix/cashout bugs6

### PR DESCRIPTION
**Pull request by Hugo0**
A **lot** of changes.

- addressLink used everywhere
- recipientInput improved
- recent account suggestions added
- tooltips fixed
- various cashout bugs fixed

TODO: no way of both suggesting IBAN AND US accounts. Need to rework this flow in the future to also suggest browser us accounts if possible.

### CODE REVIEW

File: src/app/cashout/page.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import * as components from '@/components'
+import { Cashout } from '@/components'
```
Comment (nezz0746) [inline src/app/cashout/page.tsx:1]: I like

File: src/components/Cashout/Cashout.tsx [hunk 2]
```diff
@@ -92,6 +92,43 @@ export const Cashout = ({}) => {
         fetchAndSetCrossChainDetails()
     }, [])
 
+    // For testing - check if URL has screen param
+    useEffect(() => {
```
Comment (nezz0746) [inline src/components/Cashout/Cashout.tsx:96]: Should this stil be in here if it was for testing ?

File: src/components/Global/AddressLink/index.tsx [hunk 3]
```diff
@@ -1,27 +1,37 @@
...
         setUrl(`https://debank.com/profile/${address}`)
-    }, [address])
+
+        // Update display: prefer ENS name, fallback to shortened address
+        if (ensName) {
+            setDisplayAddress(ensName)
+        } else {
+            setDisplayAddress(utils.printableAddress(address))
+        }
+    }, [address, ensName])
```
Comment (nezz0746) [inline src/components/Global/AddressLink/index.tsx:27]: nit: I feel like there's no real need for effects or state here. We're just displaying conditionally either the ens is present or not

File: src/components/Cashout/Cashout.tsx (hunk 2 above)
Comment (Hugo0) [reply src/components/Cashout/Cashout.tsx:96]: yes! Its useful
peanut.to/cashout?screen=confirm

No reason not to, way easier  for future testing

File: src/components/Global/AddressLink/index.tsx (hunk 3 above)
Comment (Hugo0) [reply src/components/Global/AddressLink/index.tsx:27]: no cuz we do:

0x --> fetch if ens exists --> ens

**Review by nezz0746 — APPROVED** [review]

**Pull request merged**